subjects/devops/grades/README.md

```diff
@@ -0,0 +1,108 @@
+## grades
+
+### Instructions
+
+Create a script `grades.sh` which will ask the user for a specific number of students in order to evaluate them. After the number is set, the script will ask you to introduce the "names" and "grades" of the persons you wish to evaluate.
```

Comment:
As phrased here it seems like the script will prompt to ask the number of users while it takes it as an argument instead. What do you think about rephrasing maybe making it more clear that the number of students will be passed as an argument to the script?

Reply:
done